plugins/ExtendedTools/objmgr.c

```diff
@@ -1207,57 +2781,160 @@ INT_PTR CALLBACK WinObjDlgProc(
 
                     if (item)
                     {
-                        if (!PhHandleCopyListViewEMenuItem(item))
+                        POBJECT_ITEM directory;
+                        ET_OBJECT_ENTRY entry = { 0 };
+
+                        PhGetTreeViewItemParam(context->TreeViewHandle, context->SelectedTreeItem, &directory, NULL);
```

Comment:
This produces a compilation error when building for release:
```
[ERROR] (1) plugins\ExtendedTools\objmgr.c(2788): error C4703: potentially uninitialized local pointer variable 'directory' used [C:\Users\jxy\source\repos\winsiderss\systeminformer\plugins\ExtendedTools\ExtendedTools.vcxproj]LINK : fatal error LNK1257: code generation failed [C:\Users\jxy\source\repos\winsiderss\systeminformer\plugins\ExtendedTools\ExtendedTools.vcxproj]
```

Needs to be this:
```suggestion
                        if (PhGetTreeViewItemParam(context->TreeViewHandle, context->SelectedTreeItem, &directory, NULL))
                        {
```